helm/deploy.sh

```diff
@@ -94,6 +94,7 @@ set -x
 # Exit with error code when command fails so that GH Action fails
 set -e
 helm upgrade "$RELEASE_NAME" "helm/$HELM_CHART" -n "${NAMESPACE}" \
+  --timeout 600 \
```

Comment:
to double-check: I'm reading this as 600 seconds (10 minutes). The PR writeup suggests a timeout parameter of 5 minutes. Do we need to adjust this value?  potentially:
```suggestion
  --timeout 300 \
```

Reply:
Sorry for the confusion, it is actually a default of 5 minutes so the comment should say 10 which is the intended value.  However, I also figured out that we are already setting the timeout in the HELM_ARGS variable that collects parameters prior to the `helm upgrade` command: https://github.com/department-of-veterans-affairs/abd-vro/blob/improve-deployment-reliability/helm/deploy.sh#L36, so this PR Is not going to help